src/isValidQuery.js

```diff
@@ -0,0 +1,6 @@
+const minMax = new RegExp(/(\()(?:[a-z][a-z]+)(-)((?:[a-z][a-z]+))?(-)?((?:[a-z][a-z]+))(:)(\s?)(\d+)((?:[a-z][a-z]+))\)/)
```

Comment:
After doing some digging, I can confirm that that the matchMedia API isn't throwing an error.

<img width="857" alt="screen shot 2017-01-25 at 7 49 24 pm" src="https://cloud.githubusercontent.com/assets/1577013/22316070/a4be17a4-e338-11e6-9117-e66cd46b1952.png">

It does look like that the package [css-mediaquery](https://github.com/ericf/css-mediaquery) does some validation though, which is what [matchMedia](https://github.com/iceddev/matchmedia) delegates to when `window.matchMedia` isn't defined.  Despite that validation, the query "poorly-formatted-query" still passes validate in css-media-query because of the RegExp used [here](https://github.com/ericf/css-mediaquery/blob/master/index.js#L14).

So maybe this should just be handled in one of those packages instead?